src/DependencyInjection/SynoliaSyliusMailTesterExtension.php

```diff
@@ -26,5 +27,9 @@ public function load(array $config, ContainerBuilder $container): void
             ->registerForAutoconfiguration(ResolvableFormTypeInterface::class)
             ->addTag(ResolvableFormTypeResolverCompilerPass::TAG_ID)
         ;
+
+        if (!class_exists('Sylius\InvoicingPlugin\SyliusInvoicingPlugin')) {
```

Comment:
You can add `use` on the top of the file.
`use Sylius\InvoicingPlugin\SyliusInvoicingPlugin;`
and here 
```suggestion
        if (!class_exists(SyliusInvoicingPlugin::class)) {
```

Reply:
i don't think so because if the sylius invoicing plugin is not installed this class has no access